src/sagemaker/jumpstart/artifacts/model_uris.py

```diff
@@ -84,14 +87,55 @@ def _retrieve_model_uri(
         sagemaker_session=sagemaker_session,
     )
 
+    model_artifact_key: str
+
     if model_scope == JumpStartScriptScope.INFERENCE:
+        instance_specific_prepacked_hosting_artifact_key: Optional[str] = (
+            model_specs.hosting_instance_type_variants.get_instance_specific_prepacked_artifact_key(
+                instance_type=instance_type
+            )
+            if instance_type
+            and getattr(model_specs, "hosting_instance_type_variants", None) is not None
```

Comment:
Isn't the name of the property `hosting_custom_instance_type_variants` in the MH PR?